debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -592,6 +592,84 @@ public static SchemaRefreshMode parse(String value) {
             .withDescription("The name of the Postgres 10+ publication used for streaming changes from a plugin." +
                     "Defaults to '" + ReplicationConnection.Builder.DEFAULT_PUBLICATION_NAME + "'");
 
+    public enum AutoCreateMode implements EnumeratedValue {
+        /**
+         * No Publication will be created, it's expected the user
+         * has already created the publication.
+         */
+        DISABLED("disabled"),
+        /**
+         * Enable publication for all tables.
+         */
+        ALL_TABLES("all_tables"),
+        /**
+         * Enable publication on a specific set of tables.
+         */
+        WHITELIST("whitelist");
```

Comment:
Actually it's whitelist + blacklist together. Perhaps "filter" or "filtered"?

Reply:
yeah filtered makes the most sense I think given the desired behaviour